### PR TITLE
Add-IncomeCommand-Test

### PR DESCRIPTION
**Pull request by teamotitan**
Added tests for income command, as well as fix a few bugs from IncomeCommand

### CODE REVIEW

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/IncomeCommand.java [hunk 1]
```diff
@@ -12,18 +14,17 @@
...
-        int feesOwed = 0;
-        int feesEarned = 0;
-        //ObservableList<Person> personList = model.getFilteredPersonList();
-        //for (Person person: personList) {
-        //fees_owed += person.getFeeOwed();
-        //fees_earned += person.getFeeEarned();
-        //}
-        return new CommandResult("Fees owed: " + feesOwed + ", fees earned: "
-                + feesEarned);
+        double totalOwed = 0;
```
Comment (Feng1231) [inline src/main/java/seedu/address/logic/commands/IncomeCommand.java:20]: Good that you changed it to double!

**Review by Feng1231 — CHANGES_REQUESTED** [review]
LGTM. But you missed out on AddressBookParser test for IncomCommand

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -88,6 +89,12 @@ public void parseCommand_list() throws Exception {
         assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
     }
 
+    @Test
+    public void parseCommand_income() throws Exception {
+        assertTrue(parser.parseCommand(IncomeCommand.COMMAND_WORD) instanceof IncomeCommand);
+        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
```
Comment (Feng1231) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:95]: line 95 looks weird. You meant to test 'income 3'?

**Review by Feng1231 — CHANGES_REQUESTED** [review]

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Review by Feng1231 — APPROVED** [review]
Nice, thanks for the changes

**Pull request merged**